tools/hw_decoder_bench.py

```diff
@@ -80,20 +87,79 @@ def RN50Pipeline():
     return images
 
 
+@pipeline_def(batch_size=args.batch_size, num_threads=args.num_threads, device_id=args.device_id,
+              prefetch_queue_depth=1)
+def EfficientnetPipeline():
```

Comment:
This pipeline stands out in this file - the others are clearly training pipelines, utilizing internal readers. This one is an oddball and uses ExternalSource. My suggestion is to mention that it's an inference pipeline to make it clear that it's unlike the other two.
```suggestion
def EfficientNetInferencePipeline():
```

Reply:
Done

Reply:
Almost ;)